third_party/packages/cupertino_icons/example/example.md

```diff
@@ -0,0 +1,3 @@
+Please find the usage of `cupertino_icons` here
```

Comment:
This should include some kind of introduction explaining that `cupertino_icons` is used via Flutter's `CupertinoIcons` class rather than directly, since it does not have its own API surface.